src/api/app/models/bs_request_permission_check.rb

```diff
@@ -124,6 +124,13 @@ def cmd_changestate_permissions(opts)
 
       check_newstate_action!(action, opts)
 
+      # TODO: Get the relevant project attribute, from the target project or target package. Retrieve the accepter and check if it's the same person than the creator. And fail if true
+      target_package = Package.get_by_project_and_name(action.target_project, action.target_package)
```

Comment:
note that target packages often don't exist. this code looks crashing in such a case

Reply:
Yeah, I noticed... Incoming push...